src/NzbDrone.Core/Notifications/Pushcut/PushcutPayload.cs

```diff
@@ -0,0 +1,13 @@
+using Newtonsoft.Json;
+
+namespace NzbDrone.Core.Notifications.Pushcut;
+
+public class PushcutPayload
+{
+    [JsonProperty(PropertyName = "title")]
+    public string Title { get; set; }
+    [JsonProperty(PropertyName = "text")]
+    public string Text { get; set; }
+    [JsonProperty(PropertyName = "isTimeSensitive")]
+    public bool? IsTimeSensitive { get; set; }
```

Comment:
Properties are automatically converted to camel case

```suggestion
    public string Title { get; set; }
    public string Text { get; set; }
    public bool? IsTimeSensitive { get; set; }
```